webpack/localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -238,7 +235,7 @@ export class LocalizationPlugin implements Webpack.Plugin {
                   assetPath.match(Constants.LOCALE_FILENAME_TOKEN_REGEX)
                 ) {
                   // Does this look like an async chunk URL generator?
-                  if (typeof options.chunk.id === 'string' && options.chunk.id.match(/^\" \+/)) {
+                  if (typeof options.chunk.id === 'string' && (options.chunk.id as string).match(/^\" \+/)) {
```

Comment:
We shouldn't need this cast. Maybe need to store `options.chunk.id` in a local `const` so the resolver can guarantee it didn't change?

Reply:
The typings here are wrong. It's marked as a `number` in the typings, so we need to explicitly cast this here because otherwise its inferred type is `never`.